app/src/main/java/com/withpersona/webview/MainActivity.java

```diff
@@ -113,9 +116,13 @@ public boolean onShowFileChooser(
         if (takePictureIntent.resolveActivity(getPackageManager()) != null) {
           // Create the File where the photo should go
           File photoFile = null;
+          // Create an image file name
+          String timeStamp = new SimpleDateFormat("yyyyMMdd_HHmmss", Locale.US).format(new Date());
+          String imageFileName = "JPEG_" + timeStamp + "_";
+          File storageDir =
+              Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES);
```

Comment:
nit: This method may be deprecated, maybe prefer `Context#getExternalFilesDir(String)` here.
https://developer.android.com/reference/android/os/Environment.html#getExternalStoragePublicDirectory(java.lang.String)